### PR TITLE
Update Video tutoriales

### PR DESCRIPTION
**Pull request by JavGuerra**
Added video «Curso de Astro, Generador de Sitios Web Estáticos» by Fazt, in Spanish.

### CODE REVIEW

File: src/content/docs/es/community-resources/content.mdx [hunk 1]
```diff
@@ -32,6 +32,7 @@ Checa los siguientes cursos y tutoriales para aprender más sobre Astro.
   - [Construye una plataforma de blog personalizada con Astro y Appwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
   - [Curso acelerado de Portfolio con Astro](https://www.youtube.com/watch?v=TwWvNK0yHjI)
   - [Construye un blog full stack con Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)
+  - [Curso de Astro, Generador de Sitios Web Estáticos](https://youtu.be/sOXW0ZnJxbQ?si=6e0MJazn3o3dEP6x)
```
Comment (sarah11918) [inline src/content/docs/es/community-resources/content.mdx:35]:
```suggestion
  - (ES) [Curso de Astro, Generador de Sitios Web Estáticos](https://www.youtube.com/watch?v=sOXW0ZnJxbQ) 
```
Let's use this format, but on the `/en/` page!

**Review by sarah11918 — COMMENTED** [review]
Thank you for adding this video @JavGuerra ! We would like to put this on the `/en/` page so that everyone can see it. 

Can you please change this PR to add this on the English page only? (not the `/es/` page).

This can be added later to the other languages in a different PR. But first, we always make content changes to the English docs. Thank you!

**Comment by JavGuerra** [comment]
> Can you please change this PR to add this on the English page only? (not the `/es/` page).

Sorry, but this video is in Spanish. Surely it will not be useful to people who do not understand the language. I see however that in the list of videos in Spanish, all of them are videos in English. This could also be corrected, or the language indicated in some way.

**Comment by sarah11918** [comment]
Hi! It is fine that the video is in Spanish for the English page! Some Spanish speakers may be reading the English docs and be interested in that content. It is not our place to assume people can only speak one language, so it is better to have all content available to everyone.

The English pages are our true sources. The language pages are all translated from that source, so no language pages have their own special content.

Our translators (volunteer, community) have a dedicated space to discuss on Discord, and each individual language decides whether they prefer to translate the titles of content into their language, or leave in the original. Some languages DO translate the titles of English content, and others do not. We let them decide individually based on what they believe is most helpful to their readers. The Spanish translators have decided to translate. This is why I suggested using `(ES)` before the title to indicate that this content is actually in Spanish. It will be helpful to the English readers AND to the Spanish readers.

If you are on Discord, you are welcome to join our Spanish language channel for translations and ask questions directly of the translators there! https://astro.build/chat `#docs-i18n`

**Comment by sarah11918** [comment]
The format is good, but reminder we need this changed in a different file! Please do not change the `es` file.

This needs to update `/en/`! 

Then, after the change is made to English, all the other sites will be marked as "needing updates" to match the English page. THEN we need this page updated in the Spanish version, too!  :smile: 

I noticed you join the Discord! If you have any questions, you can always ask in the #docs-i18n channel there!

File: src/content/docs/en/community-resources/content.mdx [hunk 2]
```diff
@@ -33,6 +33,7 @@ Check out the following courses and tutorials to learn more about Astro.
   - [Build a custom blog platform with Astro and Appwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
   - [Astro JS Portfolio Crash Course](https://www.youtube.com/watch?v=TwWvNK0yHjI)
   - [Build a full stack blog with Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)
+  - (ES) [Curso de Astro, Generador de Sitios Web Estáticos](https://www.youtube.com/watch?v=sOXW0ZnJxbQ)
```
Comment (sarah11918) [inline src/content/docs/en/community-resources/content.mdx:36]:
```suggestion
  - (ES) [Curso de Astro, Generador de Sitios Web Estáticos](https://www.youtube.com/watch?v=sOXW0ZnJxbQ)
```

**Pull request merged**